wp/sections/definition.tex

```diff
@@ -0,0 +1 @@
+Before continue, it is necessary to define some terms so that the reader correctly understands the context of what was written. The first and basic concept that should be introduced is \textbf{"rule"}. In diKTat, a “rule” is a class where it is checked if the source code of a certain part of the code style. \textbf{Ruleset}, in turn, is a set of such "rules". \textbf{Abstract syntax tree(AST)} is a tree representation of the abstract syntactic structure of source code written in a programming language. Each node of the tree denotes a construct occurring in the source code. Node \footnote{\url{ https://www.ibm.com/support/knowledgecenter/SSZHNR_2.0.0/org.eclipse.jdt.doc.isv/reference/api/org/eclipse/jdt/core/dom/ASTNode.html}} - is a a node of AST. \textbf{CICD} - continuous integration (CI) and continuous delivery (CD) is a methodology that allows application development teams to make changes to code more frequently and reliably. \textbf{KDoc} - is the language used to document Kotlin code (the equivalent of Java's JavaDoc).
```

Comment:
add "inspection" here also

Reply:
Yes, in diktat we have not only `Rule`s, but also `Warnings` which we often tend to call 'rules' or 'inspections' so that definitely needs some clarification.